validator/src/main.rs

```diff
@@ -1173,6 +1179,12 @@ pub fn main() {
                 .takes_value(false)
                 .help("When this is set to true, the system will use QUIC to send transactions."),
         )
+        .arg(
+            Arg::with_name("tpu_do_batch")
```

Comment:
I think having this is actually simpler for most users. They just need to be concerned with doing batch or non batching -- and no need to decide on a batch size which is more advanced configuration and leaving it with the system default.

Reply:
we can drive it purely on the batch size arg.  I don't see this really being "optional" after we have it tested out anyway (which is why I suggested hiding all of the settings).  Either it will work and we'll default it, or it won't and we'll remove it.

Reply:
I made that change. Thanks, it is simpler